repositories/src/test/scala/cromwell/pipeline/datastorage/dao/repository/utils/TestProjectUtils.scala

```diff
@@ -5,14 +5,15 @@ import java.util.UUID
 import cromwell.pipeline.datastorage.dto.{ Project, ProjectId, UserId }
 
 object TestProjectUtils {
-  val userUuid: String = UUID.randomUUID().toString
-  val projectUuid: String = UUID.randomUUID().toString
 
+  private def randomUuidStr: String = UUID.randomUUID().toString
+  def getDummyProjectId: ProjectId = ProjectId(randomUuidStr)
+  private def getDummyUserId: UserId = UserId(randomUuidStr)
```

Comment:
Please, do not remove `// todo` that I left in the previous comment.
This method must be moved to `TestUserUtils` and without that `todo` we can easily forget about it.